qubes/vm/appvm.py

```diff
@@ -120,6 +120,11 @@ def on_domain_loaded(self, event):
     def on_property_pre_reset_template(self, event, name, oldvalue=None):
         '''Forbid deleting template of running VM
         '''  # pylint: disable=unused-argument
+        default_template = getattr(self.app, 'default_template', None)
+        if default_template is not None:
+            raise qubes.exc.QubesValueError('Cannot unset template. '\
+                f'Use `qvm-prefs {self.name} template {default_template.name}`'\
```

Comment:
Is  naming qvm-prefs specifically here a good idea? That error may show up in a bunch of places (not necessarily in CLI), and user may prefer to use GUI. Maybe just say "You can set it to the current default (name) instead" ?

Reply:
I thought about the exact scenario. And every existing GUI tool I know uses some sort of ComboBox with predefined choices of non-blank templates. Unless there is one that I do not know. Still your choice.

Reply:
> Maybe just say "You can set it to the current default (name) instead" ?

It is done.